### PR TITLE
Jnaneshsr123 patch 1

### PR DESCRIPTION
**Pull request by jnaneshsr123**

### CODE REVIEW

File: program [hunk 1]
```diff
@@ -1,4 +1,2 @@
 hello world welcome
-welconme to the world 
-enjoy the day
-qjwdygaygdjgba
+
```
Comment (BabaBcone) [inline program:2]: comment

Comment (BabaBcone) [inline program:2]: comment

**Comment by Sravanthithv18** [comment]
hey

**Review by Sravanthithv18 — APPROVED** [review]
hello hey

**Pull request merged**